spec/Formatter/AggregateMoneyFormatterSpec.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace spec\Money\Formatter;
+
+use Money\Currency;
+use Money\Exception\FormatterException;
+use Money\Money;
+use Money\MoneyFormatter;
+use PhpSpec\ObjectBehavior;
+
+class AggregateMoneyFormatterSpec extends ObjectBehavior
+{
+    function it_is_initializable(MoneyFormatter $moneyFormatter)
+    {
+        $this->beConstructedWith([
```

Comment:
This should go into a let method (see other specs) which is the same as setUp in PHPUnit.